src/patternfly/components/Expandable/expandable.scss

```diff
@@ -0,0 +1,84 @@
+@import "../../patternfly-imports";
+
+.pf-c-expandable {
+  // Expandable
+  --pf-c-expandable--Width: #{pf-size-prem(450px)};
+  --pf-c-expandable--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-expandable--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-expandable--BoxShadow: var(--pf-global--BoxShadow--md);
+  --pf-c-expandable--PaddingTop: var(--pf-global--spacer--2xl);
+  --pf-c-expandable--PaddingRight: var(--pf-global--spacer--3xl);
+  --pf-c-expandable--PaddingBottom: var(--pf-global--spacer--2xl);
+  --pf-c-expandable--PaddingLeft: var(--pf-global--spacer--2xl);
+
+  // Title
+  --pf-c-expandable--c-title--MarginBottom: var(--pf-global--spacer--lg);
+
+  // Expandable Toggle
+  --pf-c-expandable__toggle--MarginTop: var(--pf-global--spacer--xl);
+  --pf-c-expandable__toggle--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-expandable__toggle--Color: var(--pf-global--active-color--300);
+
+  // Expandable Toggle Icon
+  --pf-c-expandable__toggle-icon--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-expandable__toggle-icon--Color: var(--pf-global--Color--100);
+
+  // Expandable Expanded
+  --pf-c-expandable__content--MarginTop: var(--pf-global--spacer--xl);
+
+  // Expandable Footer
+  --pf-c-expandable__footer--Margin-Top: var(--pf-global--spacer--2xl);
+
+  width: var(--pf-c-expandable--Width);
+  padding: var(--pf-c-expandable--PaddingTop) var(--pf-c-expandable--PaddingRight) var(--pf-c-expandable--PaddingBottom) var(--pf-c-expandable--PaddingLeft);
+  background-color: var(--pf-c-expandable--BackgroundColor);
+  background-clip: padding-box;
+  border: var(--pf-c-expandable--BorderWidth) solid transparent;
```

Comment:
I don't know that we need this. It should just display like regular content. Shouldn't need this for high contrast mode like a dropdown menu.